src/mgr/DaemonServer.cc

```diff
@@ -1626,6 +1627,7 @@ bool DaemonServer::_handle_command(
 	      }
 	      if (anm.osd != CRUSH_ITEM_NONE) {
 		pg_acting.insert(anm.osd);
+		avail_no_missing.insert(anm.osd);
```

Comment:
Here you see that _pg_acting_ AND _avail_no_missing_ is set from _q.second.avail_no_missing_ for degraded PGs.  There is no place for a degraded PG that they can diverge.

In summary:
non-degraded pg -> _pg_acting_ from _q.second.acting_
degraded pg -> _pg_acting_ AND _avail_no_missing_ identically set from q.second.avail_no_missing